### ISSUE TITLE
docker-compose with "network_mode: host"

### ISSUE THREAD

**Issue by squromiv** [b-m-f/WirtBot#197]
I added "network_mode: host" do docker-compose file. After that
![screen_2021 04 25](https://user-images.githubusercontent.com/15985877/116001390-e65b0b00-a5fc-11eb-9a7c-0ddc955700fe.png)
Ok. Deleted section with ports
![screen_2021 04 25](https://user-images.githubusercontent.com/15985877/116006806-82910c00-a615-11eb-90d6-417f70015fdf.png)
And rerun docker-compose
![screen_2021 04 25](https://user-images.githubusercontent.com/15985877/116006902-df8cc200-a615-11eb-872c-e0b048ba439e.png)
So what next?

**Comment by b-m-f** [comment]
Can you try to delete that line with `disable_ipv6`.

I think this only works when using docker networking.

If that is the case I am going to update the documentation.

**Comment by squromiv** [comment]
> Can you try to delete that line with `disable_ipv6`.

Of course, I did it. The result is unpromising. Take a look at [screenshot](http://s01.geekpic.net/di-ESYB8G.jpeg), please.

**Comment by squromiv** [comment]
> Looks like port 53 is already in use on your host machine.
Perfectly. I`ve found out, that [it is used](https://www.linuxuprising.com/2020/07/ubuntu-how-to-free-up-port-53-used-by.html) by "systemd-resolved". So we need to free it up.

Finally, in order to use "network_mode: host" in docker-compose file, we need 
1) to delete these strings:
```
ports:
    - 53:53
    - 80:80
    - 3030:3030
    - 10101:10101/udp
    sysctls:
     - net.ipv6.conf.all.disable_ipv6=0
```
2) to free up port 53, used by another service by default in Ubuntu
